providers/kind/kind.sh

```diff
@@ -26,11 +26,10 @@ declare -r KIND_DIR="${KIND_DIR:-"$PROJECT_ROOT/tmp/kind"}"
 declare -r KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-kind}
 declare -r KIND_REGISTRY_NAME=${KIND_REGISTRY_NAME:-kind-registry}
 declare -r KIND_IMAGE_REPO=${KIND_IMAGE_REPO:-localhost:5001}
+declare -r KIND_CONFIG_PATH=${KIND_CONFIG_PATH:-"$PROJECT_ROOT/providers/kind"}
 
 # constants
-declare -r KIND_CONFIG_PATH="$PROJECT_ROOT/providers/kind"
 declare -r KIND_MANIFESTS_DIR="$KIND_CONFIG_PATH/manifests"
-
 declare -r KIND_DEFAULT_NETWORK="kind"
 declare -r KIND_CONFIG_YAML="$KIND_DIR/kind.yml"
```

Comment:
Shouldn't this be the one that should be made as an ENV var - `KIND_CONFIG_PATH` ?

Reply:
@vprashar2929 I feel this change makes multiple assumptions about a kind.yaml file in KIND_CONFIG_PATH. 
```
	sed <"$KIND_MANIFESTS_DIR/kind.yml" \
		-e "s/$default_registry_name/${KIND_REGISTRY_NAME}/g" \
		-e "s/$default_registry_port/${REGISTRY_PORT}/g" \
		>"$KIND_CONFIG_YAML"

	sed <"$KIND_MANIFESTS_DIR/local-registry.yml" \
		-e "s/$default_registry_port/${REGISTRY_PORT}/g" \
		>"$KIND_REGISTRY_YAML"
```
These must exist and these assumptions must be validated. 

Moreover, to indicate a Directory, please use `_DIR`.